ci-operator/config/openshift/prometheus/openshift-prometheus-master.yaml

```diff
@@ -0,0 +1,30 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: origin-v4.0
+    namespace: openshift
+    tag: base
+canonical_go_repository: github.com/openshift/prometheus
+images:
+- from: base
+  to: prometheus
```

Comment:
If you plan to change versions and orchestrate in the 4.0.x timeframe you may eventually have to add a prometheus-2.6.  This is fine for now, just be aware.